### PR TITLE
Content display

### PR DESCRIPTION
**Pull request by Moise-code**
In this branch of our project : 

- I have retrieved api data to display on our home page.
- I have added the array of element to be used.

### CODE REVIEW

File: src/modules/displayfood.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const display = async () => {
 /* eslint-enable */
   // let markup = '';
```
Comment (Ambient567) [inline src/modules/displayfood.js:8]: I suggest that you do not commit old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history.

File: src/modules/displayfood.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ export const display = async () => {
               </div>
               <div class="column flex">
                 <button class="button">Comments</button>
+                <button class="button">Reservations</button>
             </section>`;
 
         // renderLikes(item.idMeal);
```
Comment (Ambient567) [inline src/modules/displayfood.js:42]: Same issue as discussed above. I suggest that you do not commit old pieces of code as inline comments.

File: src/modules/displayfood.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ export const display = async () => {
 /* eslint-enable */
```
Comment (Ambient567) [inline src/modules/displayfood.js:7]: We set up linters to assist us in writing standardized code. Please enable linters and try to resolve the issue. If a rule proved difficult to solve, you are only permitted to disable that rule. Even so, you'll need to determine whether the rule you're disabling is something that can be easily solved or not.

File: src/modules/food.js [hunk 4]
```diff
@@ -1,2 +1,3 @@
 /* eslint-disable */
```
Comment (Ambient567) [inline src/modules/food.js:1]: Same issue as discussed above. Please enable linters and try to resolve the issue.

**Review by Ambient567 — CHANGES_REQUESTED** [review]
Hi @Moise-code,

Good job so far! :+1: 
There are some issues that you still need to work on to go, but you are almost there!

Highlights
No linter errors :heavy_check_mark:
Good commit messages :heavy_check_mark:

Required Changes :recycle: 
Check the comments under the review.

Cheers and Happy coding! :clap: :clap: :clap:

File: src/modules/food.js (hunk 4 above)
Comment (Moise-code) [reply src/modules/food.js]: Thank you so much and everything is resolved.

**Pull request merged**